test/e2e/start_test.go

```diff
@@ -249,4 +249,71 @@ var _ = Describe("Podman start", func() {
 		Expect(session1).Should(Exit(0))
 		Expect(session1.OutputToString()).To(BeEquivalentTo(cid2))
 	})
+
+	It("podman start container sets HOME to home of caller if HOME is mounted", func() {
+		home, _ := os.UserHomeDir()
+		session := podmanTest.Podman([]string{"create", "--user", "root", "--volume", fmt.Sprintf("%s:%s:ro", home, home), ALPINE, "ls"})
+		session.WaitWithDefaultTimeout()
```

Comment:
all commands must check `Expect(session).Should(Exit(0))`, please fix this for all your commands.